### ISSUE TITLE
MacVim Plugin issues + 0.3.1

### ISSUE THREAD

**Issue by quirkey** [topfunky/PeepOpen-Issues#202]
Just updated to v0.3.1 and PeepOpen stopped respecting the current directory. Doing <Leader>p in any project would result it in indexing/opening `~/`. I tried reinstalling the plugin, no dice. To try to make it more real, I went into my ~/.vim folder and `rm plugin/peepopen.vim`. Reopened PeepOpen and tried reinstalling the plugin, doesn't seem to be replacing the rm'd file.

On the latest MacVim snapshot (57), SnowLeopard (10.6.6)

Thanks

**Comment by topfunky** [comment]
I'm looking into this right now. I wasn't able to reproduce the bug but I'll try it under other configurations.

Does your project have any of the files that PeepOpen uses to look for a project root directory? .git, Rakefile, README?

**Comment by quirkey** [comment]
Yes, all of the above. Tried on multiple different projects. All with all of the above (.git, Rakefile, README)

**Comment by quirkey** [comment]
Is there any way to get a previous version of the app? Maybe I could try going back to 0.2.5 and seeing if its a release issue or  something specific to my local

**Comment by topfunky** [comment]
I've put temporary copies of older versions here:

http://dl.dropbox.com/u/1989993/PeepOpen-0.2.4.dmg

http://dl.dropbox.com/u/1989993/PeepOpen-0.3.0.dmg

**Comment by quirkey** [comment]
Downgrading to 0.3.0 and reinstalling the plugin seemed to fix it.
